raiden/network/transport/matrix/transport.py

```diff
@@ -1202,6 +1188,35 @@ def _send_with_retry(self, queue: MessagesQueue) -> None:
         retrier = self._get_retrier(queue.queue_identifier.recipient)
         retrier.enqueue(queue_identifier=queue.queue_identifier, messages=queue.messages)
 
+    def _multicast_raw(
+        self,
+        data: str,
+        message_type: MatrixMessageType = MatrixMessageType.TEXT,
```

Comment:
I think we could go without message_type as an argument here, because we only use different to establish webRTC.

Reply:
Since this is an internal method and this has a default I think it doesn't matter that much - but I don't have a strong opinion about it.
If you don't suspect there is a possibility that one would send any kind of multicast with a different `message_type` in the future I can remove it to enhance simplicity

Reply:
If we'd ever need it we could simply add it in the future ;)

Reply:
I lacked understanding of recognising that this is a classical case of `YAGNI` :)

Reply:
Removed the argument and rebased the commit